pkg/vcs/github_client/client.go

```diff
@@ -143,6 +144,20 @@ func (c *Client) ParseHook(r *http.Request, request []byte) (vcs.PullRequest, er
 			log.Info().Str("action", p.GetAction()).Msg("ignoring Github pull request event due to non commit based action")
 			return nilPr, vcs.ErrInvalidType
 		}
+	case *github.IssueCommentEvent:
+		switch p.GetAction() {
+		case "created":
+			if strings.ToLower(p.Comment.GetBody()) == "kubechecks again" {
```

Comment:
Is this the right phrasing? A lot of similar tools would do something like `/kubechecks retry` or something like that. Alternately, we could make it configurable. Still need a good default, but that removes some pressure off of getting it perfectly right.

Reply:
honestly im coming from atlantis where do `atlantis apply/run apply` but i agree that making it configureable is a good suggestion, i'll update the PR this week and tag you